cmd/commands/commands.go

```diff
@@ -122,6 +221,10 @@ func printRespJSON(resp proto.Message) {
 
 	jsonBytesReplaced := replaceCustomData(jsonBytes)
 
+	jsonBytesReplaced = replaceChannelPoint(jsonBytesReplaced)
```

Comment:
it is not necessarily a replacement but also an amendment can you add that fact in the comment.